### PR TITLE
PODS-2346 Pensions scheme data shift

### PR DESCRIPTION
**Pull request by madhuranakate**

### CODE REVIEW

File: app/connectors/OldSubscriptionCacheConnector.scala [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+package connectors
+
+import com.google.inject.Inject
+import config.FrontendAppConfig
+import javax.inject.Singleton
+import play.api.libs.ws.WSClient
+
+@Singleton
+class OldSubscriptionCacheConnector @Inject()(
```
Comment (abhijeetardale) [inline app/connectors/OldSubscriptionCacheConnector.scala]: can we have better name?

File: app/connectors/SubscriptionCacheConnector.scala [hunk 2]
```diff
@@ -27,7 +27,7 @@ class SubscriptionCacheConnector @Inject()(
                                                override val http: WSClient
                                              ) extends CacheConnector {
```
Comment (abhijeetardale) [inline app/connectors/SubscriptionCacheConnector.scala]: behavior tests are missing for this class

File: app/connectors/OldSubscriptionCacheConnector.scala (hunk 1 above)
Comment (madhuranakate) [reply app/connectors/OldSubscriptionCacheConnector.scala]: the name describes exactly what the connector is used for. we can keep it.

File: app/connectors/SubscriptionCacheConnector.scala (hunk 2 above)
Comment (madhuranakate) [reply app/connectors/SubscriptionCacheConnector.scala]: Added behaviour tests

**Pull request merged**